doc/reference/network_sriov.md

```diff
@@ -1,17 +1,32 @@
 (network-sriov)=
 # SR-IOV network
 
-The sriov network type allows one to specify presets to use when connecting instances to a parent interface
-using sriov NICs. This allows the instance NIC itself to simply specify the `network` it is connecting to without
-knowing any of the underlying configuration details.
+<!-- Include start SR-IOV intro -->
+{abbr}`SR-IOV (Single root I/O virtualization)` is a hardware standard that allows a single network interface to appear as several virtual network interfaces in a virtualized environment.
```

Comment:
Here lets change `single network interface` to `single network card port` as in this case it is infact a physical mapping of ports to virtual interfaces.